README.md

```diff
@@ -6,7 +6,7 @@
 ## This repository contains the source code and releases for the [Creators.TF](https://creators.tf) [Community Launcher](https://creators.tf/launcher)
 
 ### Core features:
-* Launcher auto-updating *(Only via .exe and .AppImage)*
```

Comment:
This is misleading, as the non installer version is ofc also a windows executable file. So we should be more specific, how its for the windows installer version.